src/plot_api/plot_api.js

```diff
@@ -326,10 +326,6 @@ Plotly.plot = function(gd, data, layout, config) {
     // so that the caller doesn't care which route we took
     return Promise.all(gd._promises).then(function() {
         return gd;
-    }, function() {
-        // clear the promise queue if one of them got rejected
-        Lib.log('Clearing previous rejected promises from queue.');
-        gd._promises = [];
```

Comment:
I was under the impression that the 2nd argument to `Promise.all().then(, )` was simply a callback and **did not* eat the any subsequent `Plotly.plot().catch()` handlers. I was wrong.